### ISSUE TITLE
1080p support

### ISSUE THREAD

**Issue by Transisto** [groupon/greenscreen#26]
The beta chrome extension allow to stream at 1080p how can I set Greenscreen.io to 1080p ?

**Comment by frabrunelle** [comment]
+1

**Comment by bennewell** [comment]
+1

**Comment by theaccordance** [comment]
Unless you've forked and extended Greenscreen, it's agnostic of the display resolution.  If you want 1080p channels, you need to style those views with a 1080p viewport in mind, and utilize better/higher quality assets like vector graphics.  

Greenscreen is doing nothing more than loading a page with an iframe to render the URL you defined for the channel.

**Comment by Transisto** [comment]
If I ask greenscreen to display a 1080p picture it displays it at 720p and crop the rest

The chromecasts have been tried on 4k and 2560x1440 and 1080p monitors, it still display everything in 720p

**Comment by theaccordance** [comment]
Would it be possible to link the picture you're using?

**Comment by wGEric** [comment]
I'm running into this issue as well. Here is an image I used to try it: http://upload.wikimedia.org/wikipedia/commons/0/0c/Vector_Video_Standards8.svg

**Comment by Transisto** [comment]
So we can display a video in 1080p but can't with a picture (webpage). Maybe Chromecast want to make sure their UI is responsive,,, but since there is no controls this doesn't make any sense.

**Comment by theaccordance** [comment]
Haven't tested the image yet myself, but this may explain things: https://developers.google.com/cast/docs/media

tl;dr - It appears that Google is limiting the size of images
